lib/experimental/fonts/font-library/font-library.php

```diff
@@ -79,7 +136,8 @@ function wp_unregister_font_collection( $collection_id ) {
 	'slug'        => 'default-font-collection',
 	'name'        => 'Google Fonts',
 	'description' => __( 'Add from Google Fonts. Fonts are copied to and served from your site.', 'gutenberg' ),
-	'src'         => 'https://s.w.org/images/fonts/16.7/collections/google-fonts-with-preview.json',
+	// TODO: This URL needs to be updated to the wporg hosted one prior to the Gutenberg 17.6 release.
```

Comment:
Update default collection src before release.

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/58186.